### PR TITLE
API Remove environmentcheck config (moved to recipe-core) and CMS dependency

### PR DESCRIPTION
**Pull request by robbieaverill**
The environment check configuration has been moved to the [cwp-recipe-core recipe](https://github.com/silverstripe/cwp-recipe-core), and this pull request also removes any hard coupling to the CMS and Reports modules.

### CODE REVIEW

File: _config.php [hunk 1]
```diff
@@ -80,27 +80,31 @@
...
+                'ssmedia' => $assetAdminModule
+                    ->getResource('client/dist/js/TinyMCE_ssmedia.js'),
+                'ssembed' => $assetAdminModule
+                    ->getResource('client/dist/js/TinyMCE_ssembed.js'),
+                'sslinkfile' => $assetAdminModule
+                    ->getResource('client/dist/js/TinyMCE_sslink-file.js'),
+            ]);
+        $cwpEditor->insertButtonsAfter('table', 'ssmedia');
+        $cwpEditor->insertButtonsAfter('ssmedia', 'ssembed');
+    }
```
Comment (robbieaverill) [inline _config.php]: @tractorcow perhaps we should move these blocks into cwp/cwp instead, which will have a recipe-cms dependency?

Comment (tractorcow) [reply _config.php]: Yes I think so, but you only need an `silverstripe/admin` dependency, not the whole recipe.

Comment (robbieaverill) [reply _config.php]: Yeah, cool =)

Comment (robbieaverill) [reply _config.php]: Ok, I've moved all of the TinyMCE configuration to cwp/cwp here: https://github.com/silverstripe/cwp/pull/36

File: src/Report/CwpStatsReport.php [hunk 2]
```diff
@@ -15,6 +15,10 @@
 use SilverStripe\Subsites\Model\Subsite;
 use SilverStripe\Versioned\Versioned;
 
+if (!class_exists(Report::class)) {
+    return;
```
Comment (NightJar) [inline src/Report/CwpStatsReport.php]: Half of the reports featured below depend on `silverstripe/cms` being installed, which is no longer a requirement of this module. Do you think this should also be moved to `cwp/cwp` as with the TinyMCE stuff?

Comment (robbieaverill) [reply src/Report/CwpStatsReport.php]: Yep - let's move them to cwp/cwp

Comment (robbieaverill) [reply src/Report/CwpStatsReport.php]: https://github.com/silverstripe/cwp/pull/37

**Comment by robbieaverill** [comment]
I've removed the stats report class, tests and translations and put them into cwp/cwp here: https://github.com/silverstripe/cwp/pull/37

Note that e96765a is an automated pull from Transifex to ensure we don't lose any new translations in this process, it can be ignored.

**Comment by robbieaverill** [comment]
Auditor 2.0.0-beta2 tagged with the CMS dependency removed their, which allows the builds to pass here.

**Pull request merged**